src/wiktextract/extractor/en/page.py

```diff
@@ -4190,3 +4192,43 @@ def process_soft_redirect_template(
                     redirect_pages.append(title)
         return True
     return False
+
+
+def process_zh_forms_templates(
+    wxr: WiktextractContext,
+    template_node: TemplateNode,
+    base_data: WordData,
+) -> None:
+    # https://en.wiktionary.org/wiki/Template:zh-forms
+    base_data["forms"] = []
```

Comment:
Is it guaranteed that base_data["forms"] doesn't already have any data? Maybe this should be a simple check:

```
if not base_data["forms"]:
    base_data["forms"] = []
```

Just in case the situation changes in the future and someone adds code that populates base_data before process_zh_forms_templates is called.